examples/java-agent/README.md

```diff
@@ -0,0 +1,14 @@
+Builds a container image that includes the [Stackdriver Debugger Java Agent](https://cloud.google.com/debugger/docs/).
+
+To build the image:
+
+1. In `pom.xml`, replace `REPLACE-WITH-YOUR-GCP-PROJECT` with your GCP project.
+
+1. If running outside of Google Cloud Platform, you must:
+     1. [Create and download service account credentials](https://cloud.google.com/debugger/docs/setup/java#local).
+     2. Copy the credentials file into the jib `extraDirectory` location so that it is copied into the container.
```

Comment:
I think the service account credential part deserves some more attention:

1. The service account should be kept safely. It's easy to put an image in a registry with public access, then anyone can download and use it.
1. I see `extraDirectory` is set to the `agent` directory in a build output in this example. So with this setup, perhaps users need to do `mvn compile` first, copy the JSON key to `${build}/agent/opt/cdbg/gcp-svc.json` (the `agent` part already feels weird, maybe just another name), and then do `mvn package`? Also note `mvn clean` will clear up the JSON.

Reply:
+100.. a service account should _never_ be included in the container image.
if the cluster was created with the proper machine credential/scope, the service account is not needed.

Reply:
I think it's fair to say that properly obtaining and managing the service account credential is outside the scope of this example (and hence why it doesn't do it :-).  The text above does say: _If running outside of Google Cloud Platform_.  I'm happy to take suggestions for improvements.

Reply:
I'll remove this bit and link to the Stackdriver Debugger page.